app/helpers/order/order.js

```diff
@@ -0,0 +1,21 @@
+import { getOrderDescription as getOrderDescriptionFromApi } from '../api/ordapi/getOrderDescription';
```

Comment:
I think we should call the name of the file `getOrderDescription` too and perhaps it should live in the `helper/routes` folder.

Reply:
Yeah, noticed that and just renamed it!

Reply:
I created the `order` folder as I thought we may end up adding functions along the lines of `getSelectedCatalogueSolution` etc. at some point (as we talked about in our session session the other day) and I thought that fit better than routes as in terms of concepts all this stuff relates to an order.

Personally, I think I'd rather leave it where it is but you know this code base better than I do so I don't mind moving it. What do you think?

Reply:
I understand where you are coming from with naming the folder `order`. However, the helper folder is organised by function rather than domain at the moment. I want to avoid having a mixture or both since that might confuse us to where things should live. But if we feel that structuring by domain may be better then let's create a tech debt to tackle this.

Reply:
Or maybe these are best placed within the file where the call to get the orderDescription from ordapi is done? what do you think?

Reply:
I've gone with the routes directory for now. I thought it might be better to keep the stuff in api for pure API stuff, if that works for you too?

Reply:
yeah that's fine. I think it's worth revisiting this at some point though.